ocs_ci/ocs/machine.py

```diff
@@ -1059,3 +1059,42 @@ def wait_for_ready_replica_count_to_reach_expected_value(
         res = False
 
     return res
+
+
+def wait_for_current_replica_count_to_reach_expected_value(
+    machine_set, expected_value, timeout=180
+):
+    """
+    Wait for the current replica count to reach an expected value
+
+    Args:
+        machine_set (str): Name of the machine set
+        expected_value (int): The expected value to reach
+        timeout (int): Time to wait for the current replica count to reach the expected value
+
+    Return:
+        bool: True, in case of the current replica count reached the expected value. False otherwise
+
+    """
+    current_replica_count = get_replica_count(machine_set)
+    log.info(f"Current replica count = {current_replica_count}")
+    log.info(
+        f"Wait {timeout} seconds for the current replica count to reach the "
+        f"expected value {expected_value}"
+    )
+    sample = TimeoutSampler(
+        timeout=timeout,
```

Comment:
Is 180 seconds sufficient in case of VMware

Reply:
Do you suggest increasing the timeout?

Reply:
I have increased the timeout to 360 seconds.